RadarSDK/RadarDelegateHolder.m

```diff
@@ -86,4 +87,15 @@ - (void)didUpdateToken:(NSString *)token {
     [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo message:[NSString stringWithFormat:@"📍 Radar token updated | token = %@", token]];
 }
 
+- (void)didUpdateLocationPermissionStatus:(RadarLocationPermissionStatus*)status {
+    if (self.delegate) {
+        [self.delegate didUpdateLocationPermissionStatus:status];
+    }
+    
+    NSDictionary *statusDict = [status dictionaryValue];
+    
+    NSString *jsonString = [RadarUtils dictionaryToJson:statusDict];
+    [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo message:[NSString stringWithFormat:@"📍 Radar location permissions updated | status = %@", jsonString]];
```

Comment:
Can you attach for QA purposes screenshots of these logs in the dashboard?

Reply:
<img width="771" alt="image" src="https://github.com/radarlabs/radar-sdk-ios/assets/139801512/bec31518-3674-4873-b95d-b88d5a34c7ae">